app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -481,6 +484,10 @@ class HomeFragment : Fragment(), CoroutineScope, AccountObserver {
                         }
                     }
                 }
+                launch(Dispatchers.Main) {
```

Comment:
Please remove this dispatcher override here. Once this is done, I think we're ready to merge.

Reply:
Oops missed this one. Got it!